crates/chain/src/spk_txout_index.rs

```diff
@@ -270,6 +270,7 @@ impl<I: Clone + Ord> SpkTxOutIndex<I> {
         self.spk_indices.get(script)
     }
 
+    // TODO: (@leonardo) Should this also be updated to return `(bitcoin::Amount, bitcoin::Amount)` instead of (u64, u64)
```

Comment:
same as above, what do you folks think ? do you see any problem in tackling this change as well ?
cc @notmandatory @ValuedMammal @storopoli

Reply:
I don't see an issue in changing this to `Amount`.